pkg/testutils/metrics.go

```diff
@@ -69,12 +69,25 @@ func IsManifestRequeueReasonCountIncreased(ctx context.Context, requeueReason, r
 	}
 	re := regexp.MustCompile(
 		metrics.MetricRequeueReason + `{requeue_reason="` + requeueReason +
-			`",requeue_type="` + requeueType +
-			`"}` + ` (\d+)`)
+			`",requeue_type="` + requeueType + reMatchCount)
 	count, err := parseCount(re, bodyString)
 	return count >= 1, err
 }
 
+func IsManifestReconcileDurationCountNonZero(ctx context.Context, manifestName string) (bool,
+	error,
+) {
+	bodyString, err := getMetricsBody(ctx)
+	if err != nil {
+		return false, err
+	}
+	re := regexp.MustCompile(
+		metrics.MetricReconcileDuration + `_count{` + metrics.MetricLabelModule +
+			`="` + manifestName + reMatchCount)
+	count, err := parseCount(re, bodyString)
+	return count >= 0, err
```

Comment:
`count > 0`, and the count in this case should be non int because I assume it would be 0. something

Reply:
good catch. The count of a histogram metric is an integer